modules/learn/pages/clusters-and-availability/xdcr-conflict-logging-feature.adoc

```diff
@@ -0,0 +1,541 @@
+   "isDeleted": false,
+   "nodeId": "127.0.0.1",
+   "revSeqno": 2,
+   "scope": "_default",
+   "xattrs": {
+     "_mou": "",
+     "_sync": "",
+     "_vv": ""
+   }
+ },
+ "tgtDoc": {
+   "bucketUUID": "6cd3a35ba2d337d84d4fcd30c3a46fb9",
+   "cas": 1749453999840231400,
+   "clusterUUID": "d976db26a5ce512a243db65a3026073f",
+   "collection": "_default",
+   "expiry": 0,
+   "flags": 0,
+   "id": "tgt_1749454000005461807_e2f416ed317beb4f16a3587656b3d3450e1bec893cabebd3b9bb857d63824da4",
+   "isDeleted": false,
+   "nodeId": "18.206.194.145",
+   "revSeqno": 2,
+   "scope": "_default",
+   "xattrs": {
+     "_mou": "",
+     "_sync": "",
+     "_vv": "{\"cvCas\":\"0x00003dca244f4718\",\"src\":\"ntR6bcR9VHcjcPpQ5osEiw\",\"ver\":\"0x00003dca244f4718\"}"
+   }
+ },
+ "timestamp": "2025-06-09T07:26:40.005Z"
+}
+----
+
+Some of the key-value pairs in CRD are as follows:
+
+|===
+| Key | Value
+
+| `timestamp`
+| Exact date and time of the event in UTC iso-8601 format:
+YYYY-MM-DDThh:mm:ss:SSSZ
+
+The timestamp will be the wall clock of the node that noted the conflict.
+
+| `id`/`srcDoc.id`/`tgtDoc.id`
+| CRD document ID prefixed with `crd_`.
+
+A copy of conflict source document ID prefixed with `src_`.
+
+A copy of conflict target document ID prefixed with `tgt_`.
+
+| `docId`
+| document id of the actual document in conflict.
+
+| `replId`
+| Replication ID.
+
+| `srcDoc` and `tgtDoc`
+| Document ID of the “copy” of source document to look up in the conflict collection and the “copy” of the target document to look up in the conflict collection.
+
+| `nodeId`
+| Node identification, which can be the hostname or IP address and port.
+
+| `bucketUUID`
+| Source/target bucket unique ID.
+
+| `cas`
+| Source/target document CAS.
+
+| `clusterUUID`
+| Source/target cluster unique ID.
+
+| `isDeleted`
+| True or false for - is the source/target document deleted?
+
+| `scope` and `collection`
+| Name of the scope and collection where the conflict source/target documents exist.
+
+| `revSeqno`
+| Source/target document’s revision sequence number.
+
+| `expiry`
+| Source/target document’s expiry time.
+
+| `flags`
+| Source/target document’s flag.
+
+| `xattrs`
+| System xattributes.
+
+| `_vv`
+| Document version vector.
+HLV metadata.
+|===
+
+[#conflict-logger]
+=== Logger
+
+The Logger in XDCR records conflict events detected during a replication.
+It operates as an asynchronous component designed to minimize the impact on replication performance and system resources.
+
+Conflict logging is attempted on the best-effort basis.
+If any error occurs, due to resource issues or a delay in writing to the conflict log collection, XDCR may pause or stop the logging process, report the error type for the pipeline, and continue with replication so that there is no High Availability issue.
+
+By default, the Conflict Logger is tuned to handle a low percentage of conflicts.
```

Comment:
Line 227. "By default, the Conflict Logger is tuned to handle a low percentage of conflicts."

should be moved to the bottom of this section "Conflict Logger"

The section called "Tunable Replication Settings" should be renamed to "REST API Conflict Log Settings", and it should be right after this section "Conflict Logger".